OemPkg/OemConfigPolicyCreatorPei/OemConfigPolicyCreatorPei.c

```diff
@@ -0,0 +1,256 @@
+/** @file
+  This module receives the static platform data for a platform and creates a config policy. It is intended to run
+  before a platform level Silicon Policy Creator that maps this config policy to silicon policy.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <Uefi.h>
+
+#include <Ppi/ReadOnlyVariable2.h>
+#include <Ppi/Policy.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseMemoryLib.h>
+#include <Library/PeiServicesLib.h>
+#include <Library/DebugLib.h>
+#include <Library/MemoryAllocationLib.h>
+#include <Library/ConfigVariableListLib.h>
+
+typedef enum {
+  KNOB_PowerOnPort0,
+  KNOB_MAX
+} KNOB;
+
+typedef struct {
+  INTN    GetCount;
+  INTN    SetCount;
+} KNOB_STATISTICS;
+
+typedef BOOLEAN (KNOB_VALIDATION_FN)(
+  CONST VOID *
+  );
+
+typedef struct {
+  KNOB                  Knob;
+  CONST VOID            *DefaultValueAddress;
+  VOID                  *CacheValueAddress;
+  UINTN                 ValueSize;
+  CONST CHAR8           *Name;
+  UINTN                 NameSize;
+  EFI_GUID              VendorNamespace;
+  INTN                  Attributes;
+  KNOB_STATISTICS       Statistics;
+  KNOB_VALIDATION_FN    *Validator;
+} KNOB_DATA;

```

Comment:
This definition will be in the standard definition from config?

Reply:
Yep, this was missing a commit, fixed.